.github/workflows/push_rockspec.yml

```diff
@@ -33,15 +33,13 @@ jobs:
 
       # https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions
       - name: Set env
-        run: echo "GIT_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+        run: echo "GIT_TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
 
       - name: Push release rockspec
         run: |
-          sed \
-            -e "s/branch = '.\+'/tag = '${GIT_TAG}'/g" \
-            -e "s/version = '.\+'/version = '${GIT_TAG}-1'/g" \
-            ${{ env.ROCK_NAME }}-scm-1.rockspec > ${{ env.ROCK_NAME }}-${GIT_TAG}-1.rockspec
-          sed -i "s/local VERSION = 'scm-1'/local VERSION = '"${GIT_TAG}"-1'/g" metrics/init.lua
+          sed -i "s/local VERSION = 'scm-1'/local VERSION = '"${env.GIT_TAG}"-1'/g" metrics/init.lua
```

Comment:
In tarantool/graphql I decided to get rid of the rockspec revision. You can look at motivation and implementation here: https://github.com/tarantool/graphql/commit/0b4ec593a9677d010ec52d9c5b49852050884155.